### PR TITLE
155 remastered about us

### LINKED ISSUE #155: Remastered About-Us
## Description
As a stakeholder I want to have a professional About-us-page in order to give my customers a nice first impression.

## Acceptance criteria
- [x] The design on the about-us-page is on par with the welcome-page.
- [x] The images on both the welcome and about-us page should scale appropriately:
  - [x] The text should be centered even on larger monitors
  - [x] The page should not scroll just to show more of the image, it should scroll if the content needs to be scrolled
- [x] Make the register-button larger and more clickable

### PR DESCRIPTION
**Pull request by XwayTomy**
## Summary
Visual upgrade to welcome- and about-us page. The text is now centered on both pages and scales according to the window size. Furthermore, the registration button is now enlarged to make it more easy to click. 

https://user-images.githubusercontent.com/73949081/169372883-6ef69b34-4cb5-44ec-8089-077a7619f81f.mp4

## Definition of done
- [x] All acceptance criteria are completed.
- [x] The code has (at least) 80% branch test coverage.
- [x] All tests pass.
- [x] The code is reviewed and accepted by (at least) two other team members.
- [x] Views adhere to design style.

Closes #155

### CODE REVIEW

File: src/Client/Pages/AboutUs.razor.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿.about-us-container {
+    background-image: url(images/pexels-fauxels-3184418.jpg);
+    padding: 0 !important;
+    margin-left: 0 !important;
+    margin-right: 0 !important;
+    margin-top: 0 !important;
```
Comment (martinjonsson01) [inline src/Client/Pages/AboutUs.razor.css:6]: nit:
Are these `!important`s necessary now that you're not restyling the Bootstrap `container` class and instead using your own `about-us-container`? If not, then it's best to not use them.

File: src/Client/Pages/AboutUs.razor.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    min-width: 100%;
+    min-height: calc(100vh - 3.5rem);
+    background-size: cover;
+    align-items: center;
+}
+
+.about-us-container h1 {
+    color: #fbebac;
```
Comment (martinjonsson01) [inline src/Client/Pages/AboutUs.razor.css:14]: nit:
Is there any reason you chose to not use any of our predefined colors here? Using `var(--prodigo-yellow)` would give you the official color.

File: src/Client/Pages/AboutUs.razor.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    text-shadow: 0 0 1rem black;
+    text-align: center;
+    margin-top: 10vh;
+}
+
+.about-us-container h4 {
+    margin-top: 3rem;
+    margin-bottom: 3rem;
+    color: #ffffff;
```
Comment (martinjonsson01) [inline src/Client/Pages/AboutUs.razor.css:23]: nit:
Is there any reason you chose to not use any of our predefined colors here? Using `var(--prodigo-white)` would give you the official color.

File: src/Client/Pages/AboutUs.razor.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    background-color: rgba(62, 75, 101, 0.75);
+    padding: 2.5rem;
+    border-radius: 2rem;
+    text-align: center;
+    max-width: 70%;
+}
+
+.aboutUsImage {
```
Comment (martinjonsson01) [inline src/Client/Pages/AboutUs.razor.css:31]: nit:
The usual naming convention is kebab case, so that'd make this name be `about-us-image`.

Comment (martinjonsson01) [reply src/Client/Pages/AboutUs.razor.css:31]: But, is this class even used anymore now that you're using `background-image`?

File: src/Client/Pages/AboutUs.razor.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    height: 100vh;
+    width: 100vw;
+    object-fit: cover;
+}
+
+.container {
+    padding: 0 !important;
+    margin: 0 !important;
+}
```
Comment (martinjonsson01) [inline src/Client/Pages/AboutUs.razor.css:40]: Is this a remnant from the old styling?

File: src/Client/Pages/WelcomePage.razor.css [hunk 6]
```diff
@@ -1,40 +1,25 @@
...
-}
-
-.container img {
-    display: flex;
-    justify-content: center;
+.welcome-container {
+    background-image: url("https://prodigo.se/wp-content/uploads/2021/06/iStock-951514270.jpg");
+    padding: 0 !important;
+    margin-left: 0 !important;
+    margin-right: 0 !important;
```
Comment (martinjonsson01) [inline src/Client/Pages/WelcomePage.razor.css:5]: nit:
Same here, I don't believe you need the `!important`s anymore!

File: src/Client/Pages/WelcomePage.razor.css [hunk 7]
```diff
@@ -1,40 +1,25 @@
...
-    padding-right: 0;
-    left: 0;
-    height: auto;
-    max-width: 100vw;
-    justify-content: center;
-    align-items: center;
-    overflow-x: hidden;
-    overflow-y: hidden;
+.welcome-container h1 {
+    color: #fbebac;
```
Comment (martinjonsson01) [inline src/Client/Pages/WelcomePage.razor.css:13]: nit:
Is there any reason you chose to not use any of our predefined colors here? Using `var(--prodigo-yellow)` would give you the official color.

**Review by martinjonsson01 — APPROVED** [review]
Magnificent 👍

File: src/Client/Pages/AboutUs.razor.css (hunk 1 above)
Comment (XwayTomy) [reply src/Client/Pages/AboutUs.razor.css:6]: They were not important after the changing the way we do things.

File: src/Client/Pages/AboutUs.razor.css (hunk 2 above)
Comment (XwayTomy) [reply src/Client/Pages/AboutUs.razor.css:14]: Good catch! Fixed.

File: src/Client/Pages/AboutUs.razor.css (hunk 5 above)
Comment (XwayTomy) [reply src/Client/Pages/AboutUs.razor.css:40]: Yes!

File: src/Client/Pages/WelcomePage.razor.css (hunk 6 above)
Comment (XwayTomy) [reply src/Client/Pages/WelcomePage.razor.css:5]: That's correct!

**Review by Matte02 — APPROVED** [review]
LGTM👍

**Pull request merged**